envoy-control-tests/src/main/resources/envoy/config_ads_disabled_endpoint_permissions.yaml

```diff
@@ -0,0 +1,67 @@
+admin:
+  access_log_path: /dev/null
+  address:
+    socket_address: { address: 0.0.0.0, port_value: 10000 }
+dynamic_resources:
+  lds_config: {ads: {}}
+  cds_config: {ads: {}}
+  ads_config:
+    api_type: GRPC
+    grpc_services:
+      envoy_grpc:
+        cluster_name: envoy-control-xds
+node:
+  cluster: test-cluster
+  id: test-id
+  metadata:
+    ads: true
+    ingress_host: "0.0.0.0"
+    ingress_port: 5001
+    egress_host: "0.0.0.0"
+    egress_port: 5000
+    use_remote_address: true
+    access_log_enabled: false
+    resources_dir: "/etc/envoy/extra"
+    proxy_settings:
+      incoming:
+        enabled: false
```

Comment:
This property doesn't exists. I assume that you added it just to show that permissions are really disabled.
But it can create wrong impression that you have to add something to disable endpoint permissions.

Maybe change 
```yaml
      incoming:
        enabled: false
```

to a comment:

```
# no "incoming" permissions defined, so access to a service is open for all.
```

to stress that incoming permissions are disabled *by default*

Reply:
Ok, I'll do that :)